### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by cabbiepete**
This is still useful as reference so I updated the dependancies and node versions to test against so its a better reference.

### CODE REVIEW

**Comment by ryan-roemer** [comment]
@cabbiepete -- Thanks for chipping in! Can you apply this CI update diff to your PR and see if that fixes what's ailing Travis / Sauce Labs?

https://github.com/ryan-roemer/backbone-testing/compare/chore-updateTravis.diff

**Comment by cabbiepete** [comment]
@ryan-roemer Travis complaining with "No binary for Chrome browser on your platform" for sauce runner tests. Any ideas?

File: .travis.yml [hunk 1]
```diff
@@ -9,15 +12,20 @@ before_script:
...
-    - secure: FiticJKWOlcxW4ze9h2sfRjY7RYAgnws87KMzCPEOrduAvpJFqCoGIsnHa7ZAL8LgLSiZOAl68R7lL4R6/lP9BHomqLTygj80ir4ix4HVf1ejIsbOUHxRxu/W+XbiTe5CPphrQhdOydISs2942mey/QtM+2l1w+V7mEGqdVg6s4=
+    - SAUCE_USERNAME=backbone-testing
+    - BROWSER_STACK_USERNAME=ryanroemer2
+    # BROWSER_STACK_ACCESS_KEY
+    - secure: "NFY7urPFNJa2uS25vqSl4YSvp/Ux5vIN6KjJwH3UF6ptwPXBqdYk39aGxoIY2jq5Pm4APzxGOmkSMGWex/+4HrwvKyCW/cFOX6ENxLT2lBb72JEHpkDroZNYPW2zyNsiKDZNqki/XfvuPeNHNOM7Rb3HMPS7xOQJWc8hGO0gJbU="
   matrix:
-    - BB_TESTING_CHECK=1
-    - BB_TESTING_KARMA=ci
-    - BB_TESTING_KARMA=sauce
-    - BB_TESTING_KARMA=bs
```
Comment (ryan-roemer) [inline .travis.yml]: L17-L19 need to be reverted. Only L20 (`BB_TESTING_KARMA=bs`) is replaced with new L20-L22.

**Review by ryan-roemer — CHANGES_REQUESTED** [review]
Have some travis fixes that may help...

**Pull request closed without merging**